Products/CMFPlone/browser/contact_info.py

```diff
@@ -67,12 +67,13 @@ def send_message(self, data):
 
         try:
             # This actually sends out the mail
-            host.send(
+            host.secureSend(
```

Comment:
I have updated the code to add a Header instead of using secureSend. See 0eac3fef65a82c7ab2171585bf0c53d1c4af9748